### PR TITLE
Fix responsive issue

### LINKED ISSUE #69: [Bug] Details panel's title is not responsive:
The details panel's title is not responsive:

Capture d’écran 2020-08-13 à 14 26 01

The expected result should be everything on a unique line with an ellipsis on the asset name. Ex:
Capture d’écran 2020-08-13 à 14 28 43

### PR DESCRIPTION
**Pull request by maeldebon**
Fixes #69

### CODE REVIEW

File: src/app/business/common/components/detail/index.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ class Detail extends Component {
         return (
             <PanelWrapper>
                 <PanelTop className={css`justify-content: space-between;`}>
-                    <Title item={item} />
+                    <Title item={item} className={css`text-overflow: ellipsis; overflow: hidden;`} />
```
Comment (jmorel) [inline src/app/business/common/components/detail/index.js:40]: Could you move this to a const?

**Comment by jmorel** [comment]
It looks like this isn't working as specified in the issue: the text is cropped with no ellispsis and the buttons disappeared.
Capture d’écran 2020-08-13 à 16 43 15

File: src/app/business/common/components/detail/index.js (hunk 1 above)
Comment (maeldebon) [reply src/app/business/common/components/detail/index.js]: I moved it directly in the "Title" component, this it is only used here.

**Comment by maeldebon** [comment]
> It looks like this isn't working as specified in the issue: the text is cropped with no ellispsis and the buttons disappeared.
> Capture d’écran 2020-08-13 à 16 43 15

Yes indeed, it worked for me when I committed, but it looks like it was due to the test I made when using the "Inspect Element" tool.

**Pull request merged**